### ISSUE TITLE
Question regarding predictions above 1 or below 0

### ISSUE THREAD

**Issue by timoleistner** [liugangcode/torch-molecule#21]
I've trained both, GNN and GREA models to predict 'activity' of molecules against a certain organism.
For training I've used SMILES accompanied by binary data (active/inactive) and the trained models predict values mostly between 1 and 0 but sometimes there are predictions above 1 or below 0. Why is that? What is the prediction type? Is it a probability or are these logits which I still have to apply a softmax function to?

**Comment by timoleistner** [comment]
Since the authors seem to have abandoned the package I'll just answer myself. Seems to be logits since the models use the PyTorch function `BCEWithLogitsLoss`.
